core/src/test/java/com/graphhopper/routing/util/CarFlagEncoderTest.java

```diff
@@ -651,4 +652,34 @@ public void testIssue_1256() {
         assertEquals(2.5, encoder.getFerrySpeed(way), .1);
         assertEquals(.5, lowFactorCar.getFerrySpeed(way), .1);
     }
+
+    @Test
+    public void testAcceptsTurnRelation() {
```

Comment:
This should go into `OSMTurnRelationTest` instead. Its very simple, just replacing `acceptsTurnRelation` calls with `isVehicleTypeConcerned..`.

Reply:
Ok I moved the test and changed the call.